harness/features/allVariables.cloud.test.ts

```diff
@@ -79,7 +72,7 @@ describe('allVariables Tests - Cloud', () => {
                 }
                 const userResponse = await createUser(url, user)
                 const userLocation = userResponse.headers.get('Location')
-                const response = await callAllVariablesCloud(clientId, url, userLocation)
+                const response = await testClient.callAllVariables(userLocation, true)
```

Comment:
how come you didnt add close to these ones?

Reply:
@ajwootto cloud doesnt have the close method